ext/bg/js/anki-note-builder.js

```diff
@@ -34,7 +34,7 @@ class AnkiNoteBuilder {
             deckName: modeOptions.deck,
             modelName: modeOptions.model,
             options: {
-                duplicateScope: 'deck'
+                duplicateScope: options.anki.duplicateScope
```

Comment:
Just checking, is the default changed to `collection` on purpose?

Reply:
This is intentional; I decided to make 'deck' an opt-in behaviour since it is different from the previous default.